common/config_parser/config_parser.cpp

```diff
@@ -0,0 +1,412 @@
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("InventoryPollIntervalSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->inventory_poll_interval_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("RetryPollIntervalSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->retry_poll_interval_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("RetryPollCount");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->retry_poll_count = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("StateScriptTimeoutSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->state_script_timeout_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("StateScriptRetryTimeoutSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->state_script_retry_timeout_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("StateScriptRetryIntervalSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->state_script_retry_interval_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("ModuleTimeoutSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->module_timeout_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	/* Vectors/arrays now */
+	e_cfg_value = cfg_json.Get("ArtifactVerifyKeys");
+	if (e_cfg_value) {
+		const json::Json value_array = e_cfg_value.value();
+		const json::ExpectedSize e_n_items = value_array.GetArraySize();
+		if (e_n_items) {
+			for (size_t i = 0; i < e_n_items.value(); i++) {
+				const json::ExpectedJson e_array_item = value_array.Get(i);
+				if (e_array_item) {
+					const json::ExpectedString e_item_string = e_array_item.value().GetString();
+					if (e_item_string) {
+						const string item_value = e_item_string.value();
+						if (count(
+								this->artifact_verify_keys.begin(),
+								this->artifact_verify_keys.end(),
+								item_value)
+							== 0) {
+							this->artifact_verify_keys.push_back(item_value);
+							applied = true;
+						}
+					}
+				}
+			}
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("Servers");
```

Comment:
Remember to implement [this conditional](https://github.com/mendersoftware/mender/blob/28bc08ac0ed6816d96372d1c7ff50dc8d03f7a1d/conf/config.go#L185-L197).